src/utils/constants.ts

```diff
@@ -106,6 +107,130 @@ const CHAIN_CONFIGS: { [key: number]: ChainConfig } = {
   },
 }
 
+export const WRAPPED_NATIVE_CURRENCY: { [chainId in ChainId]: Token } = {
+  [ChainId.MAINNET]: new Token(1, '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2', 18, 'WETH', 'Wrapped Ether'),
```

Comment:
surely we already have this mapping defined somewhere? if not, I think it should probably live in sdk-core https://github.com/Uniswap/sdk-core/blob/main/src/addresses.ts

Reply:
So this mapping usually lives in smart order router, and so our applications all read it from there. There's usually no concept of "real" tokens at the SDK layer so I was hesitant on adding it to SDK-core. This was what I was talking about before about having some "data" SDK. 

Though I do agree that it doesnt really fit well here so I'd be open to moving it into SDK core

Reply:
OK removing this actually since we don't need to reference the wrapped version of native currencies - it is the responsibility of the caller to provide both ETH as the tokenIn and WETH as the route tokenIn